src/js/components/NavBar.js

```diff
@@ -66,18 +66,28 @@ class NavBar extends Component {
 
   render() {
 
-    const {loggedInUser, history, chunks, chapterNum, location, kanban}= this.props;
-    let searchBar=''
+    const {loggedInUser, history, chunks, chapterNum, location, kanban, chapterPage}= this.props;
+    const searchBar = QueryString.parse(location.search);
     let menu = '';
+    let book ='';
+    let chapter='';
+    let goToChapters = '';
     if (kanban) {
-      searchBar = QueryString.parse(location.search);
+      chapter =`Chapter ${chapterNum}`;
+        book = searchBar.bookName;
+        goToChapters = () => window.history.back();

```

Comment:
I ran into this yesterday but didn't say anything about it. Here's my question. Can we do something more that just history.back()?
because it creates an issue if you have used the next chapter button and now you're on a chapter other than chapter 1. So if you're on chapter 2 it doesn't redirect you back to the chapters page, but actually back to chapter 1

Reply:
yes, we could do something like history.push(/chapters?projectId=${projectId}&&bookName=${bookName}) , I'll fix it

I did that because we were using the old components for the chapters page so I did not want to attached to props from those components. so it was the hacky way

but now we can fix it, thanks you catch that!